src/test/java/co/rsk/federate/signing/hsm/client/PowHSMSigningClientBtcTest.java

```diff
@@ -379,14 +380,14 @@ private static HSMSignature createMockSignature() {
     private static List<Arguments> legacyPegoutArgProvider() {
         List<Arguments> arguments = new ArrayList<>();
 
-        int[] hsmVersions = {2, 4, 5};
-        for (int hsmVersion : hsmVersions) {
+        HSMVersion[] hsmVersions = {HSMVersion.V2, HSMVersion.V4, HSMVersion.V5};
```

Comment:
Would it make sense to have a method in the enum that returns PowHSM versions?

Reply:
Done